rfc-136-remove-our-backup-cdn-in-gcp.md

```diff
@@ -0,0 +1,70 @@
+# Remove the backup CDN in Google Cloud Platform
+
+## Summary
+
+We have two backup content delivery networks (CDNs), in case Fastly have an outage.
+The first is AWS CloudFront, the second is Google Cloud CDN.
+
+A simultaneous outage of both Fastly and AWS CloudFront is very unlikely. So
+the Google Cloud CDN backup provides very little extra reliability.
```

Comment:
It might be worth illustrating the probability of two CDNs being down? Both AWS CloudFront's SLA and Fastly's SLA (depending on our account) offer a discount if we get less than 99.99% availability (53 minutes of downtime/service/year). Does running two CDNs give us 6 nines (~30s of simultaneous downtime)? If so, running an additional CDN (presumably with an extra four nines) seems excessive, it'd probably take us longer than 30s to failover, and there are bigger problems affecting GOV.UK's availability.

Reply:
Added: https://github.com/alphagov/govuk-rfcs/pull/136/files#diff-dec23adcf70b72a15b0c38431315d7edecbbe25c093cdcc7b225fa2b95ee6d68R91-R109